cmd/neofs-cli/modules/container/create.go

```diff
@@ -39,6 +40,23 @@ It will be stored in sidechain when inner ring will accepts it.`,
 		placementPolicy, err := parseContainerPolicy(containerPolicy)
 		common.ExitOnErr(cmd, "", err)
 
+		key := key.GetOrGenerate(cmd)
+		cli := internalclient.GetSDKClientByFlag(cmd, key, commonflags.RPC)
+
+		if !force {
+			var prm internalclient.NetMapSnapshotPrm
+			prm.SetClient(cli)
+			resmap, err := internalclient.NetMapSnapshot(prm)
+			if err != nil {
+				common.ExitOnErr(cmd, "unable to get netmap snapshot: %w", err)
+			}
+			var nm = resmap.NetMap()
+			_, err = nm.ContainerNodes(*placementPolicy, nil)
+			if err != nil {
+				common.ExitOnErr(cmd, "could not build container nodes based on given placement policy: %w", err)
```

Comment:
do we need any comment about the `--force` flag in the error msg here? to make it easier for a user to find out about the new flag?

Reply:
Added more info in error message.